src/shared/components/mutationTable/column/ChromosomeColumnFormatter.tsx

```diff
@@ -4,12 +4,12 @@ import { Mutation } from 'cbioportal-ts-api-client';
  * @author Selcuk Onur Sumer
  */
 export default class ChromosomeColumnFormatter {
-    public static getSortValue(data: Mutation[]): number | null {
-        const chromosome = ChromosomeColumnFormatter.getData(data);
+    public static getSortValue(data: Pick<Mutation, 'chr'>[]): number | null {
+        const chromosome = this.getData(data);
```

Comment:
Im surprised this works, I thought `this` wasnt allowed in static functions?

Reply:
it is working. I think with the class it can be accessed in non-static way